src/dc_receive_imf.rs

```diff
@@ -1035,7 +1034,7 @@ async fn add_parts(
                                 sort_timestamp,
                             )
                             .await?;
-                            return Ok(None); // do not return an error as this would result in retrying the message
+                            // do not return an error as this would result in retrying the message
```

Comment:
That's a semantic change what happens when setting protection failed:
Before: Only an info message "Cannot set protection: …" was shown.
Now: An info message "Protection en|disabled" _and_ an info message "Cannot set protection: …" was shown.

To me, both seems equally good, so I went with the one that was easier to implement, but I can also add the code to not add an info message on errors.

Reply:
If I understand correctly, protection enabled/disabled is a replacement message that goes into message bubble, but the message bubble itself goes into trash, so there is no visible difference.

Reply:
Ah, wait, this message does not go to trash.
I also thought about `set_msg_failed` instead of `add_info_msg`, but it seems to be only working for outgoing messages.